server/src-lib/Hasura/Cache/Bounded.hs

```diff
@@ -24,16 +33,23 @@ import qualified Data.IORef         as IORef
 import qualified Data.Vector        as V
 import           GHC.Natural        (Natural)
 
+-- MISC TODO: 
+--  - benchmark and consider unsafeLookupIncreasePriority and unsafeInsertIncreasePriorityView
```

Comment:
I can't find the definitions of these `unsafe` variants

Reply:
They are here: http://hackage.haskell.org/package/psqueues-0.2.7.2/docs/Data-HashPSQ.html#g:9

Not documented, but the underlying IntPSQ have some docs. I'd want to see whether there was test coverage for these too, and benchmark